Checkstyle error:
```
> Task :streams:checkstyleTest

[2023-01-26T22:45:54.179Z] [ant:checkstyle] [ERROR] /home/jenkins/jenkins-agent/workspace/Kafka_kafka-pr_PR-13142/streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBRangeIteratorTest.java:396:44: '{' is not followed by whitespace. [WhitespaceAround]

[2023-01-26T22:45:54.179Z] [ant:checkstyle] [ERROR] /home/jenkins/jenkins-agent/workspace/Kafka_kafka-pr_PR-13142/streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBRangeIteratorTest.java:396:45: '}' is not preceded with whitespace. [WhitespaceAround]
```

--- a/streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBRangeIteratorTest.java
+++ b/streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBRangeIteratorTest.java
@@ -393,7 +393,7 @@ public class RocksDBRangeIteratorTest {
             true,
             true
         );
-        rocksDBRangeIterator.onClose(() -> {});
+        rocksDBRangeIterator.onClose(() -> { });
         rocksDBRangeIterator.close();
         verify(rocksIterator);
     }

